test/cqlpy/cassandra_tests/validation/operations/compact_storage_test.py

```diff
@@ -17,7 +17,7 @@
 # let's skip its test.
 # See issue #3882
 @pytest.mark.skip
-def testSparseCompactTableIndex(cql, test_keyspace):
+def testSparseCompactTableIndex(compact_storage, cql, test_keyspace):
```

Comment:
Since **all** the tests in this file are about compact_storage, you could have used an autouse fixture to set this option for all of them instead of each one individually. Perhaps something like this:

```
@pytest.fixture(scope="module", autouse=True)
def all_tests_in_this_file_use_compact_storage(compact_storage):
     pass
```

Reply:
Done in v10